mat-process-api/V1/Controllers/ProcessDataController.cs

```diff
@@ -45,12 +48,28 @@ public class ProcessDataController : Controller
         [Route("{processRef}")]
         [Produces("application/json")]
         [ProducesResponseType(typeof(GetProcessDataResponse), 200)]
-        public IActionResult GetProcessData(string processRef)
+        public IActionResult GetProcessData([FromRoute] GetProcessDataRequest request)
         {
-            _logger.LogInformation($"Get ProcessData request for process ID {processRef}");
-            var request = new GetProcessDataRequest() {processRef = processRef };
-            var result = _processDataUsecase.ExecuteGet(request);
-            return Ok(result);
+            var isRequestValid = _getValidator.Validate(request);
+
+            if (isRequestValid.IsValid)
+            {
+                try
+                {
+                    _logger.LogInformation($"Get ProcessData request for process ID {request.processRef}");
```

Comment:
The logger should come before the validation call so we have a log that the request for process ref X has occured

Reply:
Moved logger to run first